regulations/static/regulations/js/source/views/comment/comment-review-view.js

```diff
@@ -58,11 +55,13 @@ var CommentReviewView = Backbone.View.extend({
       dataType: 'json'
     });
     $xhr.then(this.previewSuccess.bind(this));
+    this.previewLoading = true;
+    this.render();
   },
 
   previewSuccess: function(resp) {
-    this.mode = 'submit';
-    this.previewUrl = resp.url;
+    window.location = resp.url;
```

Comment:
That's a good point re. shadiness, but I'm not sure what else we can do here. I tried opening the link in a new tab instead:

```javascript
window.open(resp.url, '_blank');
```

But that triggered a pop-up warning in chrome that users might not notice and accept.

Also, do any modern browsers *not* open a download dialog on a response with a content-disposition of attachment?